src/interfaces/api/site.ts

```diff
@@ -500,6 +500,10 @@ export class EditSite {
   @Transform(({ value }) => toUndefined(value), { toPlainOnly: true })
   @Expose()
   blocked_instances: Option<string[]>;
+  @Transform(({ value }) => toOption(value), { toClassOnly: true })
+  @Transform(({ value }) => toUndefined(value), { toPlainOnly: true })
+  @Expose()
+  taglines: Option<string[]>;
```

Comment:
Needs to be added to CreateSite too, they should be mostly identical.

Reply:
I thought that we decided against adding to CreateSite?

Reply:
Its fine for now I spose, but IMO every field should match, otherwise we'll have to maintain and remember differences.